### DISCUSSION TITLE
Is this still alive? Are there forks with active development?

### DISCUSSION THREAD

**Discussion by oyatek** [prevwong/craft.js#642] [Q&A] [answered]
Looks like a great start, but it seems there hasn't been any activity lately. Does anyone know if there are forks with active development?

**Comment by qu8n** [comment, marked as answer]
The latest feature commit was [5 days ago](https://github.com/prevwong/craft.js/commits/main/), so I'm not sure what you're referring to.

The maintainer has also been working on a major update. You can find the details [here](https://github.com/prevwong/craft.js/issues/507).

**Reply by alihammad99** [reply]
All the best :)

**Comment by oyatek** [comment]
Oh, my bad! Sorry for the oversight. Thanks for the link to the update details discussion.
